docs/getting_started/install_manually.md

```diff
@@ -385,6 +391,29 @@ If Composer asks you for your token, you must log in to your GitHub account and
 
 Please note that a clean install of eZ Platform doesn’t include the DemoBundle anymore.
 
+!!! enterprise
+
+    ###### Enable Date-based Publisher
+
+    To enable delayed publishing of Content using the Date-based publisher, you need to set up cron to run commands periodically.
```

Comment:
There is only one command that needs to run periodically ( but there multiple commands needed in order to set it up)
I would change it into:

(...) set up cron to run the command `app/console ezstudio:scheduled:publish` periodically.